client/src/request.js

```diff
@@ -1,26 +1,30 @@
 const photos = (productId) => {
-  return fetch(`http://localhost:4002/photos/id/${productId}`)
+  return fetch(`http://ec2-3-136-203-39.us-east-2.compute.amazonaws.com:4002/photos/id/${productId}`)
   .then(res => res.json())
-  .then((productPhotos) => {
+  .then(productPhotos => {
     return productPhotos;
-    // this.setState({
-    //   productId: productId,
-    //   primaryPhotoUrl: productPhotos.primaryUrl,
-    //   productPhotosUrls: productPhotos.productUrls
-    // });
   })
   .catch(error => {
     let productPhotos = ['https://via.placeholder.com/640x480?text=Unable+to+get+primary+product+image'];
     for (let i = 2; i <= 5; i++) {
       productPhotos.push(`https://via.placeholder.com/640x480?text=Unable+to+get+product+image+${i}`);
     };
     return productPhotos;
-    // this.setState({
-    //   productId: null,
-    //   primaryPhotoUrl: 'https://via.placeholder.com/640x480?text=Unable+to+get+primary+product+image',
-    //   productPhotosUrls: productPhotos
-    // });
   });
 };
 
-export default { photos };
\ No newline at end of file
+const productInfo = (productId) => {
+  return fetch((`http://localhost:4004/description/${productId}`))
```

Comment:
You can update this url from local to: http://ec2-18-217-85-161.us-east-2.compute.amazonaws.com:4004/description/${productId}

Reply:
Thank you for the heads up!